README.adoc

```diff
@@ -177,6 +177,9 @@ The Vulkan Guide is designed to help developers get up and going with the world
 
 // include::{chapters}common_pitfalls.adoc[]
 
+== xref:{chapters}hlsl.adoc[Using HLSL shaders]
```

Comment:
so you also need to add this to the `guide.adoc` file (for the time being)

Reply:
I see. That's the "whole guide in one place" document. I have added the new chapter in there too.

Reply:
ya, if you run the `make` it builds a cool single-page guide, the goal is in the future to host that more properly somewhere if we would want